src/main/java/dobby/Dobby.java

```diff
@@ -0,0 +1,57 @@
+package dobby;
+
+import dobby.commands.Command;
+
+import java.util.Scanner;
+import java.io.IOException;
+
+
+/**
+ * Dobby is a chat-bot that helps user track their daily tasks.
+ */
+public class Dobby {
+    private static Scanner scanner = new Scanner(System.in);
+    private static DobbyList dobbyList;
+    private static UserInput ui;
+    private static String filePath;
+
+    /**
+     * Constructor for Dobby class.
+     *
+     * @param filePath Path of file to be used for storing tasks.
+     */
+    @SuppressWarnings("checkstyle:EmptyCatchBlock")
+    public Dobby(String filePath) {
+        dobbyList = new DobbyList();
+        ui = new UserInput();
+        this.filePath = filePath;
+        DobbyStorage.load(dobbyList, filePath);
+    }
+
+    /**
+     * Starts the program.
+     *
+     * @throws IOException
+     */
+    private static void dobbyStart() throws IOException {
+        DobbyChat.sayHello();
```

Comment:
It is not immediately obvious what DobbyChat is, I didnt know that it was a static method. Maybe can write comments here?